### PR TITLE
Update repo to be what is actually on dev (in auth0 web front-end)

### PR DESCRIPTION
**Pull request by davidread**
Although there is an Auth0 extension that updates Auth0 rules from this repo, I thought this wasn't set-up for dev, when I made manual changes. So this is to commit the MFA changes to dev. And also there is some whitespace stuff in another file. So next time the auto-deploy runs on dev for another change, there will be no surprise changes to these files.

### CODE REVIEW

File: rules/Multifactor-Google-Authenticator-Do-Not-Rename.js [hunk 1]
```diff
@@ -1,12 +1,26 @@
 function (user, context, callback) {
-    var AUTHENTICATOR_LABEL = 'MOJ Analytical Platform (dev)';
+    var rangeCheck = require('range_check');
+
+    var AUTHENTICATOR_LABEL = 'MOJ Analytical Platform (Alpha)';
```
Comment (xoen) [inline rules/Multifactor-Google-Authenticator-Do-Not-Rename.js]: Shouldn't this be `dev`? Was this copied from `dev`?

Comment (davidread) [reply rules/Multifactor-Google-Authenticator-Do-Not-Rename.js]: ah, it does say dev on dev, so this I must have made an error here

**Review by xoen — APPROVED** [review]
A part from the label saying "alpha" rest looks OK to me (haven't checked the IPs tho)

File: rules/Multifactor-Google-Authenticator-Do-Not-Rename.js (hunk 1 above)
Comment (davidread) [reply rules/Multifactor-Google-Authenticator-Do-Not-Rename.js]: I see that dev got auto-deployed (triggered by other changes) the old version of this a couple of days ago. But yes it said Alpha before that. I'll push a fresh version that is a copy from alpha but with this word changed.

**Pull request merged**